.SRCINFO

```diff
@@ -1,68 +1,74 @@
 pkgbase = firefox-wayland-hg
-	pkgdesc = Standalone web browser from mozilla.org (mozilla-unified hg, release branding, targeting wayland)
-	pkgver = r659341.4e0bb3e
+	pkgdesc = Standalone web browser from mozilla.org (mozilla-unified hg, nightly branding, targeting wayland)
+	pkgver = 118.0a1+20230818.2+hf01044248c85
 	pkgrel = 1
-	url = https://www.mozilla.org/firefox/
+	url = https://www.mozilla.org/firefox/channel/#nightly
 	arch = x86_64
-	license = MPL
 	license = GPL
 	license = LGPL
-	makedepends = git-cinnabar
-	makedepends = unzip
-	makedepends = zip
+	license = MPL
+	makedepends = cbindgen
+	makedepends = clang
 	makedepends = diffutils
-	makedepends = yasm
-	makedepends = mesa
+	makedepends = dump_syms
 	makedepends = imake
 	makedepends = inetutils
-	makedepends = xorg-server-xvfb
-	makedepends = autoconf2.13
-	makedepends = rust
-	makedepends = clang
-	makedepends = llvm
 	makedepends = jack
-	makedepends = nodejs
-	makedepends = cbindgen
-	makedepends = nasm
-	makedepends = python-setuptools
 	makedepends = lld
-	makedepends = dump_syms
+	makedepends = mold
```

Comment:
```suggestion
```

Reply:
Hmm, you want to remove the makedepend for mold? I kept it in there just in case in the future we can replace lld for mold.

Reply:
In the future, we can re-add it here, makedepends are never attached as explicit-installed deps anyway!

Reply:
Alright going to do that right now

Reply:
Done